lib/commons.rb

```diff
@@ -134,6 +134,6 @@ def parse_image_data_and_update_continue(response)
   private
 
   def api_get
-    WikiApi.new(CommonsWiki.new).query(@query)
+    WikiApi.new(Wiki.new(language: nil, project: 'commons')).query(@query)
```

Comment:
This should be `Wiki.get_or_create`.

Also, you'll probably want to add Commons as a fixture for the specs.

Reply:
Can I put `en` as the language in the fixture? If not, should I leave it nil, that is don't define language property at all in the fixture?

Reply:
it should probably be `commons` as the language and `wikimedia` as the project.